com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/font/Fontc.java

```diff
@@ -417,10 +417,10 @@ else if (fontDesc.getOutputFormat() == FontTextureFormat.TYPE_DISTANCE_FIELD) {
             fontMapBuilder.setSdfSpread(sdf_spread);
             fontMapBuilder.setSdfOutline(outline_edge);
             fontMapBuilder.setSdfShadow(shadow_edge);
-            fontMapBuilder.setAlpha(this.fontDesc.getAlpha());
-            fontMapBuilder.setOutlineAlpha(this.fontDesc.getOutlineAlpha());
-            fontMapBuilder.setShadowAlpha(this.fontDesc.getShadowAlpha());
         }
+        fontMapBuilder.setAlpha(this.fontDesc.getAlpha());
```

Comment:
The default value for for these values in FontMap are 1. We do need to copy these values since they are used in the font_renderer.cpp when we create a TextEntry:

https://github.com/defold/defold/blob/dev/engine/render/src/render/font_renderer.cpp#L545-L547